webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -215,6 +215,7 @@ function TeamController(props: Props) {
 
     return (
         <Switch>
+            {/* /:team already validated on parent route */}
```

Comment:
1/5, but we might want to repeat the pattern in more files so that people are more likely to copy the "correct" way of validating these